README.rst

```diff
@@ -0,0 +1,116 @@
+========
+Overview
+========
+
+.. start-badges
```

Comment:
i made it md intentionally. you can keep with that so long as setup.py is done right

see https://pypi.org/project/markdown-description-example/